tests/ops/integration_tests/saas/test_delighted_task.py

```diff
@@ -21,6 +21,9 @@ def test_delighted_connection_test(delighted_connection_config) -> None:
     get_connector(delighted_connection_config).test_connection()
 
 
+@pytest.mark.integration_saas
+@pytest.mark.integration_delighted
+@pytest.mark.asyncio
```

Comment:
@galvana can you confirm these should look like this? Seeing some failing tests and taking a stab at it

Reply:
Confirmed that when adding the markers that apply to other tests everything passes as expected

Reply:
That looks right to me! Thanks for fixing it